packages/squidex/schema/schemas/research-outputs.json

```diff
@@ -60,27 +74,94 @@
           "placeholder": "",
           "isRequired": true,
           "isRequiredOnPublish": false,
-          "isHalfWidth": false
+          "isHalfWidth": true
```

Comment:
this intended? (nothing in the ticket description which is why i ask)

Reply:
Yeah. This makes a field only take half of the page on the CMS